### DISCUSSION TITLE
Resetting the root password

### DISCUSSION THREAD

**Discussion by rabbitfang** [etcd-io/etcd#16631] [Q&A] [answered]
Is it possible to reset the root user password if it is lost? Alternatively, is there a way to disable authentication within the cluster without having a password to a user with the `root` role?

**Comment by jmhbnz** [comment, marked as answer]
Hey @rabbitfang - Thanks for your question, to the best of my knowledge I don't believe there is a way to recover an etcd user password without having another user with the permissions to auth and change the password,refer https://github.com/etcd-io/etcd/issues/9420

> We encrypt auth passwords with bcrypt, and do not support password recovery.
